packages/admin/admin/src/index.ts

```diff
@@ -11,6 +11,7 @@ export { AppHeaderMenuButton, type AppHeaderMenuButtonClassKey, type AppHeaderMe
 export { buildCreateRestMutation, buildDeleteRestMutation, buildUpdateRestMutation } from "./buildRestMutation";
 export { readClipboardText } from "./clipboard/readClipboardText";
 export { writeClipboardText } from "./clipboard/writeClipboardText";
+export { Breadcrumbs, type BreadcrumbsClassKey, type BreadcrumbsItemProps, type BreadcrumbsProps } from "./common/breadcrumbs/Breadcrumbs";
```

Comment:
Please add a changeset, if you export new components. 
We could also consider not exporting the components, as we only need them internally 🤔